myblog/blog/api/serializers.py

```diff
@@ -1,14 +1,14 @@
 from ..models import BlogPost, Comment
 from rest_framework import serializers
 
-
 class BlogPostSerializer(serializers.ModelSerializer):
     author = serializers.StringRelatedField(many=False, read_only =True)
+    tagged_users = serializers.StringRelatedField(many=True)
     
     class Meta:
         model = BlogPost
         fields = ('title', 'body', 'author', 'created_at', 'img', 'safe',
-                  'tagged_count', 'last_tag_date')
```

Comment:
Why did you removed there fields?

Reply:
was a mistake, while I was rearranging my serializer.